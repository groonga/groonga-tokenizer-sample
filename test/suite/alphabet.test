register tokenizers/sample

tokenize TokenSample "abc d e f"
