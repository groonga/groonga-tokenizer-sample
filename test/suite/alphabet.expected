register tokenizers/sample
[[0,0.0,0.0],true]
tokenize TokenSample "abc d e f"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "a",
      "position": 0
    },
    {
      "value": "b",
      "position": 1
    },
    {
      "value": "c",
      "position": 2
    },
    {
      "value": " ",
      "position": 3
    },
    {
      "value": "d",
      "position": 4
    },
    {
      "value": " ",
      "position": 5
    },
    {
      "value": "e",
      "position": 6
    },
    {
      "value": " ",
      "position": 7
    },
    {
      "value": "f",
      "position": 8
    }
  ]
]
